doc/AIRTABLE.md

```diff
@@ -0,0 +1,56 @@
+
+## Why and how are we using it?
+
+We're using AirTable because it allows non-developers (and developers!) to easily edit the data used by our app. This is very useful for:
+
+- Translations: Content designers and translators can edit the UI text used in the app by going to the translations table, doing a ctrl+f search for the English or French UI text they want to change, and then modifying it.
+- Eligibility logic: The logic used by our app to determine who is eligible for what benefits is set in the eligibilityPaths table. Each row has a unique combination of answers to the questions. Each row also has a list of benefits that are likely available to a user who answers in this way. The list of benefits are linked to rows in the benefits table.
+- Benefits: This table contains a row for each benefit, and a column (eligibilityPaths) with a list of links to rows in the eligibilityPaths table. Updating this list will update the ids in the benefits column in the eligibilityPaths table.
+- Feedback: Content submitted from the feedback bar in the app footer is written to the feedback table on AirTable. To do this, the server has a write key set in ENV variables.
+
+AirTable data is read by the server and injected into Redux, which is then sent to the client.
+
+## How does one get access?
+
+To request access to edit content in AirTable, set up an account on their website and then email vac@cds-snc.ca with your account details so we can add you to the project.
+
+## Making changes to AirTable
+
+To do anything in AirTable, always take the following steps:
+
+1. Visit the [Master AirTable base](https://airtable.com/tblHt9ZuCBwsplYw2). If you don't have permission to access it, [request permission](#how-does-one-get-access)
+2. Make some changes. See the step-by-step guides below to help you with the task you want to perform.
+3. View your changes in the live app by visiting the [data validation page](https://vac-benefits-finder.cds-snc.ca/data-validation) and clicking the "refresh cache" button at the top right.
+
+## Step by step guides
+
+### Changing UI text
+
+1. Locate the text you want to change in the [app](https://vac-benefits-finder.cds-snc.ca)
+2. Copy the text to the clipboard
+3. Navigate to the [translations sheet](https://airtable.com/tblJAMGWk4ZypG5uX/viwsHMCBUnwx1EC3f) in airtable
+4. Do a ctrl+f (windows) or command+f (mac) search on the page, and paste in the text you copied
+5. Locate the row that contains the text you want to change
+6. Enter in your new text in the English and French columns
+
+### Changing text on a benefit card
```

Comment:
Could make this just something like "Changing benefit details" and add in how to update the links.